[#109869960] Update README

dcarley opened the pull request:
### What

The version that we're using is no longer suffixed by `-dev` as it was when
we were building straight from hashicorp/terraform:master. Remove the
version from the README altogether, because there's a good chance that it
will be incorrect again in the future.

Fix some typos ("credentials" and "environment"), some grammar ("allow you
to" and "a Docker container"), and line wrap.
### How to test

Look at the README and see if it makes sense.
### Who can review

Not @dcarley

dcarley committed:
Remove version from README

The version that we're using is no longer suffixed by `-dev` as it was when
we were building straight from hashicorp/terraform:master. Remove the
version from the README altogether, because there's a good chance that it
will be incorrect again in the future.

dcarley committed:
Typo corrections in README

Fix some typos ("credentials" and "environment"), some grammar ("allow you
to" and "a Docker container"), and line wrap.

combor commented:
I don't think that information about TF_VARs is correct. It's just us using AWS keys in the output.

dcarley commented:
> I don't think that information about TF_VARs is correct. It's just us using AWS keys in the output.

Agreed. Should we change it to the un-prefixed versions `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`?

combor commented:
It's not even necessary to export these if using IAM profile. I would say - let's remove this as it depends on specific use case which auth method user wants to use.

dcarley committed:
Remove environment variables from README

These instructions were incorrect - the environment variables shouldn't be
prefixed with `TF_VAR_` if you only want to configure the AWS provider.

However we also aren't passing AWS keys like this anymore now that we're
using IAM instance profiles. So omit the instructions altogether.

dcarley commented:
ACK - removed.

combor commented on README.md:
```diff
@@ -10,8 +10,5 @@ This container allows to run Terraform ( 0.6.9-dev ) inside docker container. Yo
-        -e TF_VAR_AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID \
-        -e TF_VAR_AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY \
-        terraform
+docker run -ti terraform
```
This command fails:

```
docker run -ti terraform
Error response from daemon: No command specified
```

It should be sth like:

```
docker run -ti terraform sh
```

or 

```
docker run -ti terraform terraform -version
Terraform v0.6.9
```

dcarley committed:
Add command to docker run in README

This container has no `ENTRYPOINT` so you need to specify a command to run:

    ➜  paas-docker-terraform git:(109869960-readme) ✗ docker run -ti terraform
    Error response from daemon: No command specified
    ➜  paas-docker-terraform git:(109869960-readme) ✗ docker run -ti terraform terraform
    usage: terraform [--version] [--help] <command> [<args>]

    Available commands are:
        apply      Builds or changes infrastructure
        destroy    Destroy Terraform-managed infrastructure
        get        Download and install modules for the configuration
        graph      Create a visual graph of Terraform resources
        init       Initializes Terraform configuration from a module
        output     Read an output from a state file
        plan       Generate and show an execution plan
        push       Upload this Terraform module to Atlas to run
        refresh    Update local state file against real resources
        remote     Configure remote state storage
        show       Inspect Terraform state or plan
        taint      Manually mark a resource for recreation
        version    Prints the Terraform version

dcarley commented:
Added command - which will print the help usage.

combor commented:
Looks good. Merging.
